lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -4848,16 +4848,21 @@ namespace Js
     void FunctionBody::SetEntryToDeferParseForDebugger()
     {
         ProxyEntryPointInfo* defaultEntryPointInfo = this->GetDefaultEntryPointInfo();
-        if (defaultEntryPointInfo->jsMethod != DefaultDeferredParsingThunk && defaultEntryPointInfo->jsMethod != ProfileDeferredParsingThunk)
-        {
 #ifdef ENABLE_SCRIPT_PROFILING
+        if (defaultEntryPointInfo->jsMethod != DefaultDeferredParsingThunk
```

Comment:
Whoops- ended up fixing this already to unblock JC so you can just merge this change with my change